core/types/types.go

```diff
@@ -94,31 +94,50 @@ type Migration struct {
 type MigrationStatus int
 
 const (
+	// NoActiveMigration indicates there is currently no migration process happening on the network.
 	NoActiveMigration MigrationStatus = iota
-	MigrationNotStarted
+
+	// ActivationPeriod represents the phase after the migration proposal has been approved by the network,
+	// but before the migration begins. During this phase, validators prepare their nodes for migration.
+	ActivationPeriod
+
+	// MigrationInProgress is the phase where the migration is actively occurring. The old and new networks
+	// run concurrently, with state changes from the old network being replicated to the new network.
 	MigrationInProgress
+
+	// MigrationCompleted indicates the migration process has successfully finished,
+	// and the old network is ready to be decommissioned.
 	MigrationCompleted
+
+	// GenesisMigration refers to the phase where the node initializes with the genesis state,
+	// tries to replicate the state changes from the old network.
+	GenesisMigration
+
+	UnknownMigrationStatus
```

Comment:
Having the unknown status as the last/largest value can be problematic if we want to add another.  This could either be the zero value or set to something way ahead in the enum like 0xff.

However, I don't see anything actually setting `UnknownMigrationStatus`, just one check.  Is something missing?

Reply:
I should remove this. Not using this anywhere.